test/filesystem/file_nonmutating_test.rb

```diff
@@ -126,19 +126,19 @@ def test_file?
   include ExtraAssertions
 
   def test_dirname
-    assert_forwarded(File, :dirname, 'retVal', 'a/b/c/d') do
+    assert_forwarded(File, :dirname, 'ret_val', 'a/b/c/d') do
```

Comment:
This is OK, but the tests still pass without this change (and the next two changes). (I am not really sure why this parameter exists for this helper; it just gets compared to itself, AFAICT.)

Reply:
Yes, I realise these are just strings and they don't need to be done for rubocop... But I think they just look neater and more in keeping with everything else once all the other camel case names have been changed.